ClientTests/PingCentreTests.swift

```diff
@@ -0,0 +1,103 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+@testable import Client
+import XCTest
+import JSONSchema
+import Alamofire
+import Deferred
+import Shared
+
+private let mockTopic = PingCentreTopic(name: "ios-mock", schema: Schema([
+    "type": "object",
+    "properties": [
+        "title": ["type": "string"]
+    ],
+    "required": [
+        "title"
+    ]
+]))
+
+private var receivedNetworkRequests = [NSURLRequest]()
+
+// Used to mock the network so we don't need to rely on the interweb for our unit tests.
+class MockingURLProtocol: NSURLProtocol {
```

Comment:
Found this trick here: https://yahooeng.tumblr.com/post/141143817861/using-nsurlprotocol-for-testing. Thought it was pretty neat!